src/week02/task2/DivisibleBy3_5_15_Esra.java

```diff
@@ -0,0 +1,58 @@
+package week02.task2;
+
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class DivisibleBy3_5_15_Esra {
+
+    public static void main(String[] args) {
+
+
+        ArrayList<Integer> DivisibleBy15 = new ArrayList<>();
+        ArrayList<Integer> DivisibleBy3 = new ArrayList<>();
+        ArrayList<Integer> DivisibleBy5 = new ArrayList<>();
+
+        int n = 100;
+
+        for (int i = 1; i <= n; i++) {
+            if (i % 3 == 0) {
+                if (i % 5 == 0) {
+                    DivisibleBy15.addAll(Arrays.asList(i));
+                }
+            }if (i % 3 == 0){
+                if (i % 15 != 0) {
```

Comment:
you can remove this line and also line 28 by adding else before this if cases and if cases in line 27.